### PR TITLE
Fix importing wrong version of HTTPError exception

### PR DESCRIPTION
**Pull request by kregoslup**
Fixes https://github.com/elbuo8/sendgrid-django/issues/74

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
can you please add unit test?

**Comment by kregoslup** [comment]
OK, can I do it on the weekend?

**Comment by kregoslup** [comment]
By the way I wonder why such a small change, 3 lines, has such an impact on coverage, almost 10% decrease

**Comment by andriisoldatenko** [comment]
number of lines of code decreased :( you can find more info in https://coverage.readthedocs.io/en/coverage-4.4.2/

**Comment by dbinetti** [comment]
What is blocking this?

**Comment by andriisoldatenko** [comment]
Merging #77 into master will decrease coverage by 8.22%.

**Comment by dbinetti** [comment]
Is there a workaround?  Or did the person with the PR just abandon?

Sadly I am not skilled enough to contribute to solving the problem, but I
can say that if it doesn't actually report exceptions the library is
essentially unusable...

**Comment by kregoslup** [comment]
The problem with this is that the 3rd party library that's responsible for connecting with sendgrider is hard to mock. I will look into that further and in the meantime I can write to authors of this aforementioned library for help

File: tests/test_mail.py [hunk 1]
```diff
@@ -198,3 +220,11 @@ def test_build_sg_email_w_custom_args(self):
                  'personalizations': [{'subject': ''}],
                  'subject': ''}
             )
+            
+    def test_send_messages_error(self):
+        mock_client = MockClient(self.host)
```
Comment (kevin-brown) [inline tests/test_mail.py]: `self.host` doesn't appear to be assigned anywhere, so this test always throws an error.

**Pull request merged**